packages/components/src/Link/Link.css

```diff
@@ -0,0 +1,4 @@
+.link {
```

Comment:
@darryltec I'm not sure if Atlantis follows a class name convention? Happy to update if this is completely wrong haha.

Reply:
no strict rule on this one, link is a good addition